lab-michael/lib/bodyParser.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+  // Will parse the JSON body
+
+module.exports = function(req, cb, err) {
+  req.body = '';
+  req.on('data', function(data) {
+    req.body += data.toString();
+  });
+  req.on('end', function() {
+    req.body = JSON.parse(req.body);
+    cb(null, req.body);
+    if(err) {
+      console.error(err);
+    }
+  });
+};
+module.exports = {};
```

Comment:
This line is irrelevant. You've assigned a function to module.exports, and then reassign an empty object here. It could potentially break your code under certain instances, so best to clean that up.